client/src/main.tsx

```diff
@@ -32,6 +32,10 @@ const router = createBrowserRouter([
     path: '/sign-up',
     element: <SignUp />,
   },
+  {
+    path: '/home',
+    element: <Home />,
+  }
```

Comment:
This shouldn't be here our home component is the '/' path. You probably couldn't see it because the <PrivateRoute> component wraps it and forces you to log in using the login screen to see it otherwise it redirects you.
But yea this should be removed

Reply:
I added a comment for now that it is for testing purposes. Once we get the login working we can remove the path here.